app/src/main/java/org/greenstand/android/TreeTracker/walletselect/WalletSelectScreen.kt

```diff
@@ -30,12 +30,7 @@ import org.greenstand.android.TreeTracker.models.NavRoute
 import org.greenstand.android.TreeTracker.models.user.User
 import org.greenstand.android.TreeTracker.root.LocalNavHostController
 import org.greenstand.android.TreeTracker.root.LocalViewModelFactory
-import org.greenstand.android.TreeTracker.view.ActionBar
-import org.greenstand.android.TreeTracker.view.AppButtonColors
-import org.greenstand.android.TreeTracker.view.AppColors
-import org.greenstand.android.TreeTracker.view.ArrowButton
-import org.greenstand.android.TreeTracker.view.LocalImage
-import org.greenstand.android.TreeTracker.view.WalletUserButton
+import org.greenstand.android.TreeTracker.view.*
```

Comment:
We don't want wildcard imports. Did this help is pass CI/CD? If it builds, it should be able to pass those tests.

Reply:
It was automatic, I didn't even notice. Is that all that is left for this?? I want to make a pull request for permission management at runtime, just a few things left to tidy up on that branch.

Reply:
Please pay close attention to your PRs.